tests/controls/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+title 'RStudio User'
+
+control 'Common Users' do
+  impact 'high'
+  title 'The rstudio user should exist'
+  desc 'The rstudio user should exist This makes sure that it has the same UID & GID as the jupyter images'
+  tag 'user'
+  tag 'group'
+
+  describe user('rstudio') do
+    it { should exist }
+    its('uid') { should eq 1000 }
```

Comment:
Shouldn't this be 1001? In the Rstudio terminal I'm `davidread` (not `rstudio`), so I assume that's the one that should match up with `jovyan` in Jupyter, to have the same uid and therefore permissions to work with files in the home directory.

In rstudio:
```
rstudio:x:1000:1000::/home/rstudio:
davidread:x:1001:50::/home/davidread:
```
In Jupyter:
```
jovyan:x:1001:100::/home/jovyan:/bin/bash
```

Reply:
Yeah this is what I'm trying to figure out really. Why are we creating a new user in this container, when we:

- are not in the data science notebook
- have a user in the right groups 
- have a user we manage in the main Dockerfile

Along with that, creating a user on the fly isn't helpful for tests as `damacus` or `davidread` won't exist unless we're in the cluster. I'm not sure (yet) what value adding another user is providing here.

Reply:
Right,[ re-amended in the Jupyter Image](https://github.com/ministryofjustice/analytics-platform-jupyter-notebook/pull/42). Might be worth talking this over long term, as I still don't really understand why we're creating not using the default 1000 UIDs on both containers.